spec/Section 5 -- Validation.md

```diff
@@ -318,6 +376,19 @@ fragment multipleSubscriptions on Subscription {
 }
 ```
 
+We do not allow the `@skip` and `@include` directives at the root of the
+subscription operation. The following example is also invalid:
+
+```graphql counter-example
+subscription requiredRuntimeValidation($bool: Boolean!) {
+  newMessage @include(if: $bool) {
+    body
+    sender
+  }
+  disallowedSecondRootField @skip(if: $bool)
```

Comment:
```suggestion
```

This is not relevant for this test case, and will trigger the other validation error as well (GraphQL.js returns early I think, but that's not always the case).

Reply:
Okay. I just thought that the spec was clear enough that skip/include are not allowed on root selection sets – if they are not allowed, then no usage combination is valid. But anyway, it's not a big deal. 🙂

Reply:
You're not wrong, we are explicit that they're not allowed. I think it helps clarify why and cover the edge cases though; it seems obvious that `subscription($var: Boolean!){newMessage @skip(if:$var)}` makes no sense because it might result in zero fields in the root selection set; but if I used `@skip`/`@include` to always ensure there's exactly one field, maybe that's okay...? :no_good: